include/af/opencv2.hpp

```diff
@@ -0,0 +1,160 @@
+#pragma once
+#include <vector>
+#include <opencv2/core/core.hpp>
+#include <arrayfire.h>
+
+namespace afcv
```

Comment:
i think we should have a subnamespace instead of this.
i.e.
namespace af {
namespace opencv {
}
}